fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/Histogram.java

```diff
@@ -10,11 +10,11 @@ public class Histogram {
     private double max;
     private boolean containMax;
     private final List<Bucket> buckets;
-    private Map<Double, Long> topN;
+    private Map<Double, Long> mcv;
```

Comment:
Old stats in `_statistics_.histogram_statistics` cannot be used which can lead to stats compatible problem?

Reply:
This is an unreleased feature